feat: update TektonChain configuration

Roming22 opened the pull request:
Use the operator configuration to generate the signing-secret. Tests have shown that the operator does not overwrites an existing secret, so no additional safeguards are needed to enable the feature.

Remove the option to configure the secret name, as the name is static for TektonChain.

rh-pre-commit.version: 2.3.2
rh-pre-commit.check-secrets: ENABLED

lingyzhuang commented on installer/charts/tssc-infrastructure/templates/openshift-pipelines/tektonconfig/patch._tpl:
```diff
@@ -1,6 +1,7 @@
 {{- define "pipelines.TektonConfigPatch" -}}
 spec:
   chain:
+    generateSigningSecret: true
```
Should it be `'true'`? The same as line 5.

Roming22 replied:
It believe it can take both, I've updated for consistency.

lingyzhuang replied:
Sorry about the comment. I tested with this PR and the value should be type **bool**, not string. So please remove `''`.

Roming22 replied:
Done

Roming22 committed:
feat: update TektonChain configuration

Use the operator configuration to generate the signing-secret. Tests
have shown that the operator does not overwrites an existing secret, so
no additional safeguards are needed to enable the feature.

Remove the option to configure the secret name, as the name is static
for TektonChain.

rh-pre-commit.version: 2.3.2
rh-pre-commit.check-secrets: ENABLED

lingyzhuang commented:
/lgtm

lingyzhuang reviewed (changes requested):
Please change it back to `true`, instead of `'true'`.

Roming22 commented:
/retest

lingyzhuang commented:
/lgtm
